### ISSUE TITLE
Find-PSResource returns tags in wrong format, when searching for scripts

### ISSUE THREAD

**Issue by peetrike** [PowerShell/PSResourceGet#1240]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSResourceGet)
- [X] Search the existing issues.

### Steps to reproduce

```powershell
Find-PSResource send-passwordnotification -Repository PSGallery | select -ExpandProperty tags
```

### Expected behavior

```powershell
Find-Script send-passwordnotification -Repository PSGallery | select -ExpandProperty tags
```
```console
password
e-mail
email
notification
Windows
PSEdition_Desktop
PSEdition_Core
PSScript
```

### Actual behavior

```powershell
Find-PSResource send-passwordnotification -Repository PSGallery | select -ExpandProperty tags
```
```console
password,
e-mail,
email,
notification,
Windows,
PSEdition_Desktop,
PSEdition_Core
```

```powershell
Find-PSResource send-passwordnotification -Repository Local | select -ExpandProperty tags
```
```console
password

e-mail
email
notification
Windows
PSEdition_Desktop
PSEdition_Core
PSScript
```

### Error details

no error

### Environment data

Powershell: 5.1 and 7.3.5
PSResourceGet: 0.5.22

**Comment by peetrike** [comment]
This problem is also probably caused by comma as separator in script metadata field Tags.

**Comment by peetrike** [comment]
And afterwards `Find-PSResource` is not able to get this package metadata, like in #1220

**Comment by SydneyhSmith** [comment]
Thanks so much for all the testing @peetrike we really appreciate it! This is resolved by #1236
